lib/yuriita/search_collection.rb

```diff
@@ -41,7 +41,7 @@ def options
     end
 
     def keywords
-      query.keywords
+      query.inputs.reject{|input| input.is_a?  Query::Input}
```

Comment:
I think this is the only part that feels off to me. What do you think of keeping the `keywords` method and doing the rejection inside that method? This would give us a nice clean interface on the Query object without collaborators needing to know about the internals.

Reply:
I pushed up a small change in https://github.com/thoughtbot/yuri-ita/pull/60/commits/eeb5c2c84857e3f469dbc4f88ceb2e812bc51063 to keep the `Query#keywords` interface.